requirements_test.txt

```diff
@@ -15,3 +15,4 @@ pytest-sugar==0.9.1
 pytest-timeout==1.3.1
 pytest==3.7.2
 requests_mock==1.5.2
+freezegun==0.3.10
```

Comment:
Let's not add new dependencies for the test suite. Just mock out `dt_util.utcnow`.

Reply:
I'm always inclined to keep dependencies to a minimum (doubly so as I'm currently having fun working through how the test dependency system works for this project). However I've been bitten before by the complexities around manipulating time for testing, and where it can end up. Time is hard, so I'd caution against the mock approach, towards abstracting that mess away to be handled properly. 

Having said that, it is your project, and I'm just a visitor to these parts. I'm happy to change things if that's what you want. Just thought I should voice my reasoning and concerns first.

Reply:
@balloob  I now have a local patch that replaces freezegun with a mock patch for `dt_util.utcnow`. If that is your preference, then I'll push that change.